Microsoft.Toolkit.Uwp/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,57 @@
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Resources;
+
+namespace Microsoft.Toolkit.Uwp.Extensions
+{
+    /// <summary>
+    /// UWP specific helpers for working with strings.
+    /// </summary>
+    public static class StringExtensions
+    {
+
+        private static ResourceLoader _independentLoader;
```

Comment:
Why the change to the initialization of this?  It should be up here so it will be only initialized once on use as part of the static access contract to the member.

Reply:
```suggestion
        private static ResourceLoader _independentLoader = ResourceLoader.GetForViewIndependentUse();
```